test/controllers/backgrounds_controller_test.rb

```diff
@@ -12,29 +12,16 @@ class BackgroundsControllerTest < ActionDispatch::IntegrationTest
     assert_response :success
   end
 
-  test "should get new" do
-    get new_background_url
-    assert_response :success
-  end
-
   test "should create background" do
```

Comment:
Could you create a test "should not create background if it cannot view all samples" ?

Reply:
done